Content.Client/Interface/DreamInterfaceManager.cs

```diff
@@ -51,13 +52,13 @@ class DreamInterfaceManager : IDreamInterfaceManager {
 
             if (dmfParser.Warnings.Count > 0) {
                 foreach (CompilerWarning warning in dmfParser.Warnings) {
-                    Console.WriteLine(warning);
+                    Logger.Warning(warning.ToString());
```

Comment:
I am unsure if these should be `.ToString()` or just `.Message`

Reply:
`CompilerWarning` overrides `ToString()` to return `"Warning at " + location + ": " + Message;`. `ToString()` should be used here.